src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -471,8 +471,8 @@ public void SetRigiditySliderTooltip(float rigidity)
         var healthChangeLabel = GetNode<Label>(RigiditySliderTooltipHealthLabelPath);
         var mobilityChangeLabel = GetNode<Label>(RigiditySliderTooltipSpeedLabelPath);
 
-        float healthChange = rigidity * Constants.MEMBRANE_RIGIDITY_HITPOINTS_MODIFIER;
-        float mobilityChange = -1 * rigidity * Constants.MEMBRANE_RIGIDITY_MOBILITY_MODIFIER;
+        float healthChange = (float)((decimal)rigidity * (decimal)Constants.MEMBRANE_RIGIDITY_HITPOINTS_MODIFIER);
+        float mobilityChange = (float)(-1 * (decimal)rigidity * (decimal)Constants.MEMBRANE_RIGIDITY_MOBILITY_MODIFIER);
```

Comment:
Wouldn't `double` be enough, according to wiki decimal is 128 bit type? I don't think that kind of floating point has hardware acceleration, and I think it's rarely used so this code is a bit confusing. Double won't be confusing to anyone reading this.